code/parachain/frame/composable-traits/src/dex.rs

```diff
@@ -141,7 +141,7 @@ pub trait Amm {
 		pool_id: Self::PoolId,
 		in_asset: AssetAmount<Self::AssetId, Self::Balance>,
 		min_receive: AssetAmount<Self::AssetId, Self::Balance>,
-		keep_alive: bool,
+		keep_alive: bool, // blas: what's the meaning of this param?
```

Comment:
It's a substrate idiom with existential deposit. keep_alive == true means that a swap that causes an account to fall under the existential deposit errors.